packages/@o3r/eslint-config-otter/testing/setup-jest.builders.ts

```diff
@@ -0,0 +1 @@
+/* eslint-disable unicorn/no-empty-file -- will be deprecated in v12 */
```

Comment:
we may configure this rule globally to not check `setup-jest*.ts`
as we have a global setup relying on this file even if not needed for this packge